### DISCUSSION TITLE
Kraken accounts move to DLT Finance

### DISCUSSION THREAD

**Discussion by matijahu1** [btschwertfeger/python-kraken-sdk#252] [General]
The Kraken accounts from Germany are moving to DLT Finance. 
Do you see any issues that your trading activities are no longer supported? 
I hope you have a smooth transition! For me, it´s a complete mess because I had two accounts at Kraken.

**Comment by btschwertfeger** [comment]
Yeah that "cooperation" of Kraken is not really good for those using german accounts. As far as I understood all those news, it only affects the way assets are stored and the availability of those. So the API is not affected, but clients with a german account have to know that certain assets will no longer be available for trading and marked as "withdraw-only". Additionally trading and withdraw fees will increase. So the python-kraken-sdk is not directly affected, but all those who use it with german accounts may not be able to trade or even receive historical data of unlisted assets.

This is a big step backwards (IMHO) and Kraken will loose lots of customers who build up their wealth and infrastructure around the Kraken ecosystem. As always, there will another way of doing the *same or even better*.

**Reply by matijahu1** [reply]
I like your optimistic attitude! Personally, I´m deeply disappointed of Kraken. The unimportant small account was transferred but my main trading account not and everything is frozen. The saying "Not your keys - not your coins" is now tangible for me. But for trading, the coins have to be at the exchange. I´m still confident that I will get back my money but it takes days, weeks, maybe months.

**Reply by btschwertfeger** [reply]
I'm sorry to hear about your situation and I can understand how frustrating it can be when access to funds is restricted, especially for those used for active trading. The "Not your keys - not your coins" really hits in situations like this. Fortunately, my assets were not frozen, but I quickly removed them from Kraken. I hope yours will be released soon - Hang in there and stay positive!

**Comment by adamantmm** [comment]
Same reading here — the SDK/API surface stays valid; the breakage is account-side (asset availability + fee schedule), not request signing.

Practical checklist if you run bots against a DE account during the migration:

- treat newly withdraw-only pairs as hard disables in your universe filter; do not rely on order rejects as the signal
- re-check fee tiers after the move; maker/taker assumptions baked into sizing will silently drift
- if you held two Kraken logins, keep API keys scoped per legal entity so a key from the old account does not keep retrying dead pairs

No code change needed in python-kraken-sdk for the custody shift itself.
